assignment/api/src/resources/assignment.properties

```diff
@@ -638,7 +638,7 @@ nostudent = There is no student in the site.
 grading=Grading
 grading.no=Do not add assignment to Gradebook
 grading.gradeAssignment=Grade this assignment
-grading.send=Send grades to the Gradebook
+grading.send=Send grades to the Gradebook (only published grades will be sent)
```

Comment:
Is "published" the correct word?

grading.send=Send released grades to the Gradebook

Reply:
Perfect! Changes done.
Thanks!